src/styles.css

```
@@ -275,9 +275,9 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 .cb-infra{background:rgba(14,165,233,.07);color:#075985}
 .cb-ai{background:rgba(99,102,241,.08);color:#3730A3}
 .cb-dev{background:rgba(245,158,11,.08);color:#78350F}
-.cb-other{background:var(--border2);color:var(--muted)}
+.cb-other{background:var(--border2);color:#374151}
 [data-theme="dark"] .cb-os{background:rgba(100,116,139,.15);color:#9CA3AF}
-[data-theme="dark"] .cb-other{background:rgba(255,255,255,.05);color:var(--muted)}
+[data-theme="dark"] .cb-other{background:rgba(255,255,255,.05);color:#D1D5DB}
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)